package.json

```diff
@@ -1,7 +1,12 @@
 {
-  "author": "ReadMe <support@readme.io> (https://readme.com)",
   "name": "rdme",
+  "version": "3.8.2",
   "description": "ReadMe's API CLI",
+  "author": "ReadMe <support@readme.io> (https://readme.com)",
+  "engines": {
+    "node": "^12 || ^14 || ^16"
+  },
+  "main": "index.js",
```

Comment:
What's this line for? Just checked this and https://github.com/readmeio/rdme/pull/327 and it looks like we don't even have an `index.js` file

Reply:
ah my bad